app/src/test/java/com/codesoom/assignment/application/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,58 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.errors.InvalidAccessTokenException;
+import com.codesoom.assignment.utils.JwtUtil;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class AuthenticationServiceTest {
+
+    private AuthenticationService authenticationService;
+    private static final String SECRET ="12345678901234567890123456789010";
```

Comment:
```suggestion
    private static final String SECRET = "12345678901234567890123456789010";
```

`=` 사이에 빈 칸을 넣어주면 좋을 것 같습니다